graspy/embed/ase.py

```diff
@@ -154,5 +156,6 @@ def fit(self, graph, y=None):
         if self.diag_aug:
             A = augment_diagonal(A)
 
+        self.n_features_in_ = len(A)
```

Comment:
this can be refactored to `BaseEmbed` since it is common to both functions. same with the documentation

Reply:
going to make a new issue for this for now